src/components/VersionList/DeleteVersion.js

```diff
@@ -0,0 +1,124 @@
+import { useAlert } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import {
+    Modal,
+    ModalTitle,
+    ModalContent,
+    ModalActions,
+    ButtonStrip,
+    Button,
+} from '@dhis2/ui'
+import isEmpty from 'lodash/isEmpty'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { useLatestRelease, useUpdateVersions } from '../../hooks'
+import { androidOSVersionMin, androidOSVersionRecommended } from '../../shared'
+import { DeleteButton } from '../Button'
+
+const createVersion = (version, defaultV) => ({
+    downloadURL: version.downloadURL || defaultV.downloadURL,
+    version: version.version || defaultV.version,
+    androidOSVersion: {
+        min: version.min || androidOSVersionMin,
+        recommended: version.recommended || androidOSVersionRecommended,
+    },
+})
+
+export const DeleteVersion = ({ version, versionList, handleList }) => {
+    const release = useLatestRelease()
+    const { mutateVersion, mutateList } = useUpdateVersions()
```

Comment:
Hey, 
I notice that a `PUT` request is used to modify the versionList. Are there any tech restrictions on using a `DELETE` request instead?
Thanks!

Reply:
Hi Simona, I'm using `PUT` instead of `DELETE` request because the [Datastore](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-240/data-store.html?h=datastore+2.40#webapi_user_data_store_update_values) only allows to `GET` or `UPDATE` values inside a Key.